src/octopeer-github/main/ElementID.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Created by Youri on 08/05/2016.
+ */
+/**
+ * Class to hold elementID's so they can be a semantically different data type.
+ */
+class ElementID {
```

Comment:
There needed for the DatabaseAdaptable, ElementSelectionBehaviour and ElementEventBindingBehaviour. wich are currently all subfolders for these "types", we could possibly move them to a type folder.